framework/src/modules/chain/loader.js

```diff
@@ -159,43 +158,35 @@ class Loader {
 	 * @todo Add description for the params
 	 */
 	// eslint-disable-next-line class-methods-use-this
-	sync(cb) {
+	async sync() {
 		library.logger.info('Starting sync');
 		if (components.cache.ready) {
 			components.cache.disable();
 		}
 
 		__private.isActive = true;
 
-		async.series(
-			{
-				async calculateConsensusBefore() {
-					const consensus = await modules.peers.calculateConsensus();
-					return library.logger.debug(
-						`Establishing broadhash consensus before sync: ${consensus} %`
-					);
-				},
-				loadBlocksFromNetwork(seriesCb) {
-					return __private.loadBlocksFromNetwork(seriesCb);
-				},
-				async calculateConsensusAfter() {
-					const consensus = await modules.peers.calculateConsensus();
-					return library.logger.debug(
-						`Establishing broadhash consensus after sync: ${consensus} %`
-					);
-				},
-			},
-			err => {
-				__private.isActive = false;
-				__private.blocksToSync = 0;
+		const consensusBefore = await modules.peers.calculateConsensus();
 
-				library.logger.info('Finished sync');
-				if (components.cache.ready) {
-					components.cache.enable();
-				}
-				return setImmediate(cb, err);
-			}
+		library.logger.debug(
```

Comment:
shouldn't this be before the call to `calculateConsensus()` ? Small difference but upon potential errors might give more clues about the failure?

Reply:
We want to calculate consensus just before loading blocks from the network and just after it. So that we can see the difference before and after the sync.